docs/loki.md

```diff
@@ -0,0 +1,75 @@
+# How to run SGO with Loki
+A few examples about how to deploy with Loki for logging support.
+
+## Deploy SGO + Loki with minio for storage
+This is less resource intensive. Useful for development in crc.
+```
+ansible-playbook --extra-vars __service_telemetry_logs_enabled=true --extra-vars __deploy_minio_enabled=true run-ci.yaml
+```
+
+## Deploy SGO + Loki with OCS for storage
+This is more production like setup. It's more resource demanding and cannot be run in crc. This assumes OCS is already deployed.
+
+### Create an object bucket claim
+```
+oc apply -f - <<EOF
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: loki-s3-storage
+spec:
+  additionalConfig:
+    bucketclass: noobaa-default-bucket-class
+  generateBucketName: loki-s3-storage
+  storageClassName: openshift-storage.noobaa.io
```

Comment:
This one is fine because it's a separate object (not the ServiceTelemetry object)